backend/webserver/wittymail_server/emailapi_broker.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+import openpyxl
+import emailapi
+import re
+import util.logger as logger
+
+log = logger.get_logger(__name__)
+
+# Names of the fodder to be used to create the emails
+email_fodder_names = []
+# Fodder content used to create the emails
+email_fodder = []
+
+# Index of the emailid in the email_fodder_names[]. The emailid will be used in the 'to' field
+EMAIL_FODDER_TO_INDEX = 0 
+# Index of the emailid in the email_fodder_names[]. The emailid will be used in the 'cc' field
+EMAIL_FODDER_CC_INDEX = 0
+
+# Extended fodder names that will be appended to the fodder names provided by the user (i.e. email_fodder_names)
+extended_email_fodder_names = ['email_subject', 'email_body', 'email_attachment', 'email_sent']
+extended_email_fodder_names_EMAIL_BODY_INDEX = -3
+extended_email_fodder_names_EMAIL_SUBJECT_INDEX = -4
+extended_email_fodder_names_EMAIL_ATTACHMENT_INDEX = -2
+
+# Extended fodder that will be appended to the fodder provided by the user (i.e. email_fodder[])
+extended_default_email_fodder = [None, None, None, False]
+
+# TODO Add check to detect extention type
+def save_fodder_from_file(loc, email_fodder_names_template = None):
+    global email_fodder
+    global email_fodder_names
+   
+    _email_fodder = []  
+
+    # TODO Check if we should read wb_obj.active or wb_obj[0]
+    wb_obj = openpyxl.load_workbook(loc)   
```

Comment:
Don't we need to close this somewhere?

Reply:
Nah! (Atleast the example code somewhere didn't. Have added a TODO anyway)